src/test/bin/quic_gtest.cpp

```diff
@@ -690,6 +719,64 @@ TEST_P(WithHandshakeArgs1, ResumeRejection) {
             0);     // RandomLossPercentage
     }
 }
+
+TEST_P(WithHandshakeArgs1, ResumeRejectionByServerApp) {
```

Comment:
We didn't already have tests for synchronous rejection by the server?

Reply:
There is ResumeRejection test, which is caused by wrong ticket. This is rejected (validation failed) by OpenSSL functionality during SSL_do_handshake.
This is the case when using right ticket but server app reject